app/services/responsibility_service.rb

```diff
@@ -12,12 +12,17 @@ def to_s
   COWORKING = 'Co-Working'
   NPS = 'NPS'
 
+  # We currently don't have access to the date of the parole board decision, which means that we cannot correctly
+  # calculate responsibility for NPS indeterminate cases with parole eligibility where the TED is in the past.
+  # A decision has been made to display a notice so staff can check whether they need to override their case or not;
+  # this is until we get access to this data.
   def self.calculate_pom_responsibility(offender)
```

Comment:
Was this the gnarly "if statement" @KPobeeNorris mentioned? It is quite hard to follow as a naive reader!

Normally I'd suggest pairing to see if we can refactor it a bit, but I'd prefer to focus on getting this working and merged in, and catch it the next time we're in this area.

Reply:
Pairing on a refactor sounds good!